pkg/reconciler/buildrun/resources/sources/bundle.go

```diff
@@ -75,14 +85,21 @@ func AppendBundleStep(
 
 // AppendBundleResult append bundle source result to build run
 func AppendBundleResult(buildRun *build.BuildRun, name string, results []pipelineapi.TaskRunResult) {
-	imageDigest := findResultValue(results, fmt.Sprintf("%s-source-%s-image-digest", prefixParamsResultsVolumes, name))
+	var sourceResult = build.SourceResult{Name: name}
 
+	imageDigest := findResultValue(results, fmt.Sprintf("%s-source-%s-image-digest", prefixParamsResultsVolumes, name))
 	if strings.TrimSpace(imageDigest) != "" {
-		buildRun.Status.Sources = append(buildRun.Status.Sources, build.SourceResult{
-			Name: name,
-			Bundle: &build.BundleSourceResult{
-				Digest: imageDigest,
-			},
-		})
+		sourceResult.Bundle = &build.BundleSourceResult{Digest: imageDigest}
+	}
+
+	sourceTimestamp := findResultValue(results, fmt.Sprintf("%s-source-%s-source-timestamp", prefixParamsResultsVolumes, name))
```

Comment:
Same here, should that be in sources.go, `updateBuildRunStatusWithSourceResult` ?